### DISCUSSION TITLE
Can I help you with development by joining your team?

### DISCUSSION THREAD

**Discussion by devmansurov** [lunarphp/lunar#54] [Q&A] [answered]
I want to help you with the development with Laravel Framework for promoting your project in my free time. Can you take me on the team?

**Comment by glennjacobs** [comment, marked as answer]
Thank you for the kind offer. Unfortunately, we cannot add you to our team. However, you are more than welcome to be a contributor by suggesting improvements and submitting PRs. All I would say is if you're looking to add to or improve the system, best to check that it is likely to be accepted first.
